eclipse-theia/theia
# packages: title localization fix

#### What it does


Closes https://github.com/eclipse-theia/theia/issues/11954#issue-1476836525.

The commit fixes localization bugs found in the titles of windows.

While in the process of solving this issue, a check on other title variables found several other windows with similar localization issues. In total six issues should be fixed.


#### How to test

To test if the initial issue was solved:
1. start the application
2. open the debug-view (ex: `view` > `debug)`
3. from the debug-view in the `watch` section click `add expression` (+ icon)
4. a dialog will appear with the title `edit watch expression`
5. use the command `configure display language` (using `F1`) and install a new language
6. restart the application when prompted
7. repeat the steps 1-4 and confirm that the title of the dialog is localized

#### Review checklist


#### Reminder for reviewers


Signed-off-by: FernandoAscencio <fernando.ascencio.cama@ericsson.com>
